pkg/sensors/tracing/generickprobe.go

```diff
@@ -117,9 +121,8 @@ type genericKprobe struct {
 	// is there override defined for the kprobe
 	hasOverride bool
 
-	// reference to a stack trace map, must be closed when unloading the kprobe,
-	// this is done in the sensor PostUnloadHook
-	stackTraceMapRef *ebpf.Map
+	// global data for multi kprobe, single otherwise
+	data *genericKprobeData
```

Comment:
I don't get the `data` naming, what is the intent behind the name? Could we get something more explicit or not really?

Reply:
it's sensor specific data that needs to be present in kprobeEntry, because we need it when we process event,
so it's specific for each kprobeEntry when we use single kprobes and it's one global instance when we use kprobe multi.. I'll update the comment with this